gameover.html

```diff
@@ -0,0 +1,26 @@
+<!-- Copied this from the index.html file with minor changes. -->
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <link rel="stylesheet.css"/>
+        <title>Gatehopper - Game over screen!</title>
+        <script src="src/dependencies/p5.min.js" defer></script>
```

Comment:
I'm wondering if we need all of these references here? Will we be using anything from them at all? I was thinking the game over screen would eventually display some sort of image and then a link back to the main screen

Reply:
Fair enough, it's best to remove stuff and keep it plain and simple. I copied and pasted the index.html file and applied the title tags out of habit without considering if the references are necessary or not. In any case, I'll work on this after the meeting today.